api/src/main/scala/hmda/api/model/EditResults.scala

```diff
@@ -4,15 +4,35 @@ import hmda.validation.engine.MacroEditJustification
 
 case class LarId(loanId: String)
 case class LarEditResult(lar: LarId)
-case class EditResult(edit: String, description: String, ts: Boolean, lars: Seq[LarEditResult])
-case class EditResults(edits: Seq[EditResult])
+case class EditResult(edit: String, description: String, ts: Boolean, lars: Seq[LarEditResult]) {
+  def toCsv(editType: String) = {
+    val larCsv = lars.map(l => Seq(editType, edit, l.lar.loanId).mkString("", ", ", "\n")).mkString
+    if (ts) {
```

Comment:
Minor formatting comment: I know it's a matter of style, but I'd prefer seeing the `if`/`else` statements in this PR more condensed. Many of them could work like this:
```scala
if (ts) constructTheCsv
else larCsv
```